### PR TITLE
Stories rework: VaSkeleton

### PR DESCRIPTION
**Pull request by Alazar351**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/ui/src/components/va-skeleton/VaSkeleton.stories.ts [hunk 1]
```diff
@@ -1,12 +1,99 @@
...
+})
+
+export const Group = () => ({
+  components: { VaSkeleton, VaSkeletonGroup },
+  template: `
+  <VaSkeletonGroup class="flex gap-5">
+  <VaSkeleton />
+  <VaSkeleton />
+  <VaSkeleton />
+  <VaSkeleton />
```
Comment (m0ksem) [inline packages/ui/src/components/va-skeleton/VaSkeleton.stories.ts:84]:
```suggestion
	  <VaSkeleton />
	  <VaSkeleton />
	  <VaSkeleton />
	  <VaSkeleton />
```

File: packages/ui/src/components/va-skeleton/VaSkeleton.stories.ts [hunk 2]
```diff
@@ -1,12 +1,99 @@
...
+})
+
+export const GroupWave = () => ({
+  components: { VaSkeleton, VaSkeletonGroup },
+  template: `
+  <VaSkeletonGroup animation="wave" class="flex gap-5">
+  <VaSkeleton />
+  <VaSkeleton />
+  <VaSkeleton />
+  <VaSkeleton />
```
Comment (m0ksem) [inline packages/ui/src/components/va-skeleton/VaSkeleton.stories.ts:96]:
```suggestion
	  <VaSkeleton />
	  <VaSkeleton />
	  <VaSkeleton />
	  <VaSkeleton />
```

File: packages/ui/src/components/va-skeleton/VaSkeleton.stories.ts [hunk 3]
```diff
@@ -1,12 +1,99 @@
...
+  [Wave]<VaSkeleton animation="wave" />
+  [None]<VaSkeleton animation="none" />
+
+  `,
+})
+
+export const Delay = () => ({
+  components: { VaSkeleton },
+  template: `
+  <VaSkeleton delay=2000 animation="wave"/>
```
Comment (m0ksem) [inline packages/ui/src/components/va-skeleton/VaSkeleton.stories.ts:73]:
```suggestion
  <VaSkeleton delay="2000" animation="wave" />
```

**Review by m0ksem — APPROVED** [review]

**Review by asvae — APPROVED** [review]
Great work, thanks! 🤟

**Pull request merged**